### PR TITLE
HW6

### PR DESCRIPTION
**Pull request by Kate-Surta**
Teachers,
please find my HW6 regarding API testing in the [api_test_suit.py] file
Thank you!
CC @ericrommel @ksauhen @SSN-SPB

### CODE REVIEW

File: topics/06_api_testing/api_test_suit.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import pytest
+import requests
+from requests.exceptions import Timeout, RequestException
+from jsonschema import validate
+from unittest.mock import patch
+
+
+@pytest.fixture
+def base_url():
```
Comment (AnastasiiaKravets) [inline topics/06_api_testing/api_test_suit.py:9]: You can use fixture scope, in this case it will be good to have "session", so you don't need to mention this fixture in every test, it will be applied automatically

**Review by AnastasiiaKravets — COMMENTED** [review]
1. It will be good to have general approach to raising more user friendly exception, not just AssertionError or ValidationError, but add more info.
2. Not implemented requirements:
- Add parameterized tests to handle multiple posts.
- Use the unittest.mock library or pytest-mock to simulate API responses for edge cases, such as: API server errors; Invalid data returned by the API; Timeouts and network errors.
- Create mock tests for each CRUD operation, simulating a successful and a failed GET, POST, PUT, and DELETE requests.
